### DISCUSSION TITLE
Free NFT airdrops - Different profiles

### DISCUSSION THREAD

**Discussion by KryptoDecentral** [iota-community/Shimmer-Governance#13] [Q&A]
Hi, I would like to maximise the chances of getting lots of the free airdrops - or let me say at least one of them. Do you think its sufficient just to create lots of profiles inside my firefily wallet and deposit there everytime 1 Gi Iota tokens and the probabilites are increased?

**Comment by darkblade60** [comment]
It should be 1gi per participation in the same wallet because I can create multiple wallets with 1gi, pls reconsider this!!! I dont want to waste my time for that
